src/D2L.CodeStyle.Analyzers/ApiUsage/Events/EventHandlersDisallowedListAnalyzer.cs

```diff
@@ -10,10 +10,10 @@
 namespace D2L.CodeStyle.Analyzers.ApiUsage.Events {
 
 	[DiagnosticAnalyzer( LanguageNames.CSharp )]
-	internal sealed class EventHandlersBlacklistAnalyzer : DiagnosticAnalyzer {
+	internal sealed class EventHandlersDisallowedListAnalyzer : DiagnosticAnalyzer {

```

Comment:
I don't love `AllowedList` and `DisallowedList` but past tense felt appropriate and allow/disallow seems to be the most appropriate for these use cases. I'm open to feedback though!